.circleci/config.yml

```diff
@@ -916,6 +916,45 @@ jobs:
       - run: *gitter_notify_failure
       - run: *gitter_notify_success
 
+  t_ems_compile_ext_ens:
+    docker:
+      - image: circleci/node:10
+    environment:
+      TERM: xterm
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: Install test dependencies
+          command: |
+            sudo apt-get -qy install lsof
+      - run:
+          name: External Ens compilation
+          command: |
+            export COMPILE_ONLY=1
+            test/externalTests/ens.sh /tmp/workspace/soljson.js || test/externalTests/ens.sh /tmp/workspace/soljson.js
+
+  t_ems_test_ext_ens:
+    docker:
+      - image: circleci/node:10
+    environment:
+      TERM: xterm
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: Install test dependencies
+          command: |
+            sudo apt-get -qy install lsof
+      - run:
+          name: External Ens tests
+          command: |
+            test/externalTests/ens.sh /tmp/workspace/soljson.js || test/externalTests/ens.sh /tmp/workspace/soljson.js
+      - run: *gitter_notify_failure
+      - run: *gitter_notify_success
```

Comment:
@chriseth's comment on nightly seems to be relevant here since these are supposed to report to gitter. Otherwise, this would lead to too much notifications!

Reply:
Ah right, this would spam the channel for every push to every PR iiuc